app/services/entitlement_read_service.py

```
@@ -0,0 +1,557 @@
+
+
+# ---------------------------------------------------------------------------
+# Claim Allowance Derivation (Insurance-Specific)
+# ---------------------------------------------------------------------------
+
+
+def derive_claim_allowance(
+    entitlement_id: str,
+    class_id: str,
+    policy_config: dict,
+    reference_time_utc: datetime,
```

Ruff (0.16.0)

[warning] Unused function argument: `reference_time_utc`

(ARG001)